torchaudio/datasets/speechcommands.py

```diff
@@ -79,9 +90,30 @@ def __init__(self,
                     download_url(url, root, hash_value=checksum, hash_type="md5")
                 extract_archive(archive, self._path)
 
-        walker = walk_files(self._path, suffix=".wav", prefix=True)
-        walker = filter(lambda w: HASH_DIVIDER in w and EXCEPT_FOLDER not in w, walker)
-        self._walker = list(walker)
+        def _load_list(*filenames):
```

Comment:
Can you move this to outside for the purpose of readability?

something like `def _load_list(root_dir, *filenames)`

Reply:
thanks this looks much readable now